src/ui_txn.c

```diff
@@ -71,10 +71,15 @@ static int step_txn_type() {
 }
 
 static int step_sender() {
+  uint8_t publicKey[32];
+  cx_ecfp_private_key_t privateKey;
+  algorand_key_derive(0, &privateKey);
```

Comment:
Same here.

Reply:
Fixed (indeed `current_txn.accountId` is needed here)